seacrowd/sea_datasets/indonesian_news_dataset/indonesian_news_dataset.py

```diff
@@ -0,0 +1,142 @@
+import pickle
+from pathlib import Path
+from typing import Dict, List, Tuple
+
+import datasets
+
+from seacrowd.utils.configs import SEACrowdConfig
+from seacrowd.utils.constants import Tasks, Licenses
+from seacrowd.utils import schemas
+
+_CITATION = """\
+@misc{andreaschandra2020,
+  author    = {Chandra, Andreas},
+  title     = {Indonesian News Dataset},
+  year      = {2020},
+  howpublished = {Online},
+  url       = {https://github.com/andreaschandra/indonesian-news},
+  note      = {Accessed: 2024-02-13},
+}
+"""
+
+_DATASETNAME = "indonesian_news_dataset"
+
+_DESCRIPTION = """\
+An imbalanced dataset to classify Indonesian News articles. 
+The dataset contains 5 class labels: bola, news, bisnis, tekno, and otomotif. 
+The dataset comprises of around 6k train and 2.5k test examples, with the more prevalent classes 
```

Comment:
Please remove the trailing white spaces here. 
Run make check_file=seacrowd/sea_datasets/malaysia_tweets/malaysia_tweets.py to check the necessary changes to your script.